mvn install / mvn package fails

On Java 8, cloned down the repo and then did `mvn install`, fails:

```
[ERROR] Failed to execute goal on project hive2-shims: Could not resolve dependencies for project com.amazon.emr:hive2-shims:jar:5.2.0-SNAPSHOT: Failed to collect dependencies at org.apache.hive:hive-exec:jar:2.3.0 -> org.apache.calcite:calcite-core:jar:1.10.0 -> org.pentaho:pentaho-aggdesigner-algorithm:jar:5.1.5-jhyde: Failed to read artifact descriptor for org.pentaho:pentaho-aggdesigner-algorithm:jar:5.1.5-jhyde: The following artifacts could not be resolved: org.pentaho:pentaho-aggdesigner-algorithm:pom:5.1.5-jhyde (absent): Could not transfer artifact org.pentaho:pentaho-aggdesigner-algorithm:pom:5.1.5-jhyde from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [datanucleus (http://www.datanucleus.org/downloads/maven2, default, releases), glassfish-repository (http://maven.glassfish.org/content/groups/glassfish, default, disabled), glassfish-repo-archive (http://maven.glassfish.org/content/groups/glassfish, default, disabled), apache.snapshots (http://repository.apache.org/snapshots, default, snapshots), central (http://repo.maven.apache.org/maven2, default, releases), conjars (http://conjars.org/repo, default, releases+snapshots)] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :hive2-shims
```

```
maven-default-http-blocker (http://0.0.0.0/): Blocked
```

I tried to downgrade to `mvn` 3.6.3, I then hit:

```
[ERROR] Failed to execute goal on project hive2-shims: Could not resolve dependencies for project com.amazon.emr:hive2-shims:jar:5.2.0-SNAPSHOT: Failed to collect dependencies at org.apache.hive:hive-exec:jar:2.3.0 -> org.apache.calcite:calcite-core:jar:1.10.0 -> org.pentaho:pentaho-aggdesigner-algorithm:jar:5.1.5-jhyde: Failed to read artifact descriptor for org.pentaho:pentaho-aggdesigner-algorithm:jar:5.1.5-jhyde: Could not transfer artifact org.pentaho:pentaho-aggdesigner-algorithm:pom:5.1.5-jhyde from/to conjars (http://conjars.org/repo): Transfer failed for http://conjars.org/repo/org/pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde/pentaho-aggdesigner-algorithm-5.1.5-jhyde.pom: Connect to conjars.org:80 [conjars.org/54.235.127.59] failed: Connection timed out (Connection timed out) -> [Help 1]
```

`Connect to conjars.org:80 [conjars.org/54.235.127.59] failed`

--- a/shims/hive2-shims/pom.xml
+++ b/shims/hive2-shims/pom.xml
@@ -20,6 +20,12 @@
             <groupId>org.apache.hive</groupId>
             <artifactId>hive-exec</artifactId>
             <version>${hive2.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.pentaho</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.amazon.emr</groupId>

